packages/worker/src/api/controllers/global/configs.ts

```diff
@@ -280,13 +281,39 @@ export async function save(ctx: UserCtx<Config>) {
   }
 }
 
+function enrichOIDCLogos(oidcLogos: OIDCLogosConfig) {
+  if (!oidcLogos) {
+    return
+  }
+  oidcLogos.config = Object.keys(oidcLogos.config).reduce(
```

Comment:
Can we guarantee that `oidcLogos.config` will always be an object?

I recently fixed a null-pointer where I had to do `Object.keys(x || {})`

Reply:
It's unlikely to ever not be an object once its been created, but definitely better safe than sorry